client/lib/plans/features-list.js

```diff
@@ -445,7 +445,7 @@ export const FEATURES_LIST = {
 	[ constants.FEATURE_ADVANCED_DESIGN ]: {
 		getSlug: () => constants.FEATURE_ADVANCED_DESIGN,
 		getTitle: () =>
-			i18n.translate( '{{strong}}Advanced{{/strong}} Design Customization', {
+			i18n.translate( '{{strong}}Advanced{{/strong}} design customization', {
```

Comment:
:ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.